examples/stats/web_client_monitoring/app.js

```diff
@@ -0,0 +1,131 @@
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * An Express application to receive the web metrics and send to Stackdriver
+ * with the opencensus API.
+ */
+
+"use strict";
+
+const express = require("express");
+const assert = require('assert');
+const process = require("process");
+const bodyParser = require('body-parser');
+const { Stats, MeasureUnit, AggregationType } = require('@opencensus/core');
+const { StackdriverStatsExporter } = require('@opencensus/exporter-stackdriver');
+
+const app = express();
+app.use(express.static("web_client"));
+app.use(bodyParser.json());
+
+// The project id must be provided. First, try the local environment
+const project = process.env.GOOGLE_CLOUD_PROJECT;
+assert(typeof project !== 'undefined' && project,
+  "Please set environment variable GOOGLE_CLOUD_PROJECT to the project id.");
+console.log(`Sending metrics data to project: ${project}`);
+
+// OpenCensus setup
+const stats = new Stats();
+const exporter = new StackdriverStatsExporter({projectId: project});
+stats.registerExporter(exporter);
+const mDNSTimeMs = stats.createMeasureDouble("web_client/dnsTime",
+                       MeasureUnit.MS,
+                       "DNS lookup time");
+const mConnectTimeMs = stats.createMeasureDouble("web_client/connctTime",
+                       MeasureUnit.MS,
+                       "TLS negotiation time");
+const mLatencyMs = stats.createMeasureDouble("web_client/totalTime",
+                       MeasureUnit.MS,
+                       "Total page loading time");
+const mClickCount = stats.createMeasureInt64("web_client/clickCount",
+                       MeasureUnit.UNIT,
+                       "Number of clicks");
+const tagKey = "client";
+const dnsTimeView = stats.createView(
+  "webmetrics/dnstime",
+  mDNSTimeMs,
+  AggregationType.DISTRIBUTION,
+  [tagKey],
+  "The distribution of the DNS lookup time",
+  // Bucket Boundaries:
+  [0, 25, 50, 75, 100, 200, 400, 600, 800, 1000, 2000, 4000, 6000]
+);
+const connectTimeView = stats.createView(
+  "webmetrics/connecttime",
+  mConnectTimeMs,
+  AggregationType.DISTRIBUTION,
+  [tagKey],
+  "TLS negotiation time",
+  // Bucket Boundaries:
+  [0, 25, 50, 75, 100, 200, 400, 600, 800, 1000, 2000, 4000, 6000]
+);
+const latencyView = stats.createView(
+  "webmetrics/latency",
+  mLatencyMs,
+  AggregationType.DISTRIBUTION,
+  [tagKey],
+  "The distribution of the latencies",
+  [0, 25, 50, 75, 100, 200, 400, 600, 800, 1000, 2000, 4000, 6000]
```

Comment:
For this distribution, please take a look at https://github.com/census-instrumentation/opencensus-specs/blob/master/stats/HTTP.md#units which recommends using
```
[0, 1, 2, 3, 4, 5, 6, 8, 10, 13, 16, 20, 25, 30, 40, 50, 65, 80, 100, 130, 160, 200, 250, 
300, 400, 500, 650, 800, 1000, 2000, 5000, 10000, 20000, 50000, 100000]
```

Reply:
I have changed code according to this suggestion in the previous commit.